app/views/options.html

```diff
@@ -31,9 +31,87 @@
     </label>
     <p>Example: {{10 * options.velocityUnit().mult|longnum}} meat/{{options.velocityUnit().name}}</p>
   </div>
+<div>
+	<button id="login" style="display:none">Log in with Dropbox</button>
+	<a id="logout" href="#" style="display:none">Log out</a>
+	<button id="dbsave" style="display:none">save to Dropbox</button>
+	<button id="dbreset" style="display:none">reset Dropbox</button>
+	<script>
+		//if (window.location.href.indexOf('http://') === 0 && window.location.host.indexOf('127.0.0.1') !== 0) {
+			//window.location = window.location.href.replace('http://', 'https://');
+		//}
+
+		// Be sure to use your own app key so you can set up your own redirect URI.
```

Comment:
Looks like some of this is from https://github.com/dropbox/ClickTheBox-js/blob/master/index.html ? Please add a comment saying where you got it.

(I think their Apache license is compatible with swarmsim's GPL license; copying should be safe)